tls/s2n_cipher_suites.c

```diff
@@ -157,26 +177,30 @@ const struct s2n_record_algorithm s2n_record_alg_chacha20_poly1305 = {
      * Give it a distinct 1.2 nonce value in case this changes.
      */
     .flags = S2N_TLS12_CHACHA_POLY_AEAD_NONCE,
+    .encryption_limit = 0,
 };
 
 /* TLS 1.3 Record Algorithms */
 const struct s2n_record_algorithm s2n_tls13_record_alg_aes128_gcm = {
     .cipher = &s2n_tls13_aes128_gcm,
     .hmac_alg = S2N_HMAC_NONE, /* previously used in 1.2 prf, we do not need this */
     .flags = S2N_TLS13_RECORD_AEAD_NONCE,
+    .encryption_limit = S2N_TLS13_MAXIMUM_BYTES_TO_ENCRYPT,
 };
 
 const struct s2n_record_algorithm s2n_tls13_record_alg_aes256_gcm = {
     .cipher = &s2n_tls13_aes256_gcm,
     .hmac_alg = S2N_HMAC_NONE,
     .flags = S2N_TLS13_RECORD_AEAD_NONCE,
+    .encryption_limit = S2N_TLS13_MAXIMUM_BYTES_TO_ENCRYPT,
```

Comment:
You might want to name this something more specific, that indicates it's the aes-gcm limit. Maybe define it here instead of in _parameters.h.